### PR TITLE
Added link toGithub repo

### PR DESCRIPTION
**Pull request by PohSayKeong**

### CODE REVIEW

**Comment by PohSayKeong** [comment]
It seems like the code for the two icons are pretty long in the svg, and the GitHub one does not automatically redirect for some reason, but the Fossasia one does.

**Comment by niccokunzmann** [comment]
1. Yo should claim the google codein task for this and comment in the issues that you want to do it. 
1. The black area is what can not be seen. It should be placed in the visible area.
2. Find an SVG version of the image. It will be of better resolution and less size.

**Comment by PohSayKeong** [comment]
Hi, I can only claim a task at a time so I am unable to do that. But if I place the icons in the game itself, should it disappear with the start screen?

**Comment by niccokunzmann** [comment]
Yes, it should. If it is in the same layer as the start button, then it disappears automatically.

**Comment by PohSayKeong** [comment]
Hi, I've moved both logos into the mainscreen, they are also in .svg format now.

**Comment by niccokunzmann** [comment]
Please pull the gh-pages branch, so I can merge.

**Comment by PohSayKeong** [comment]
Ok sure, it's taking some time, will come back to you when it's done.

**Comment by PohSayKeong** [comment]
Okay,it's done.

**Comment by niccokunzmann** [comment]
Can I make a suggestion: Read [this](https://github.com/fossasia/flappy-svg/issues/7) and follow the process described in the task "[Game Flappy-SVG: Impement improvements](https://codein.withgoogle.com/dashboard/tasks/5378521694208000/)".
PS: if you want to do me a favor, tell me if the link "[Game Flappy-SVG: Impement improvements](https://codein.withgoogle.com/dashboard/tasks/5378521694208000/)" works, so I know if students can use the same link as mentors.

**Comment by PohSayKeong** [comment]
Oh no it doesn't, it says I don't have access to the page.

**Comment by niccokunzmann** [comment]
Can you search for this name and give me the link?

**Comment by PohSayKeong** [comment]
https://codein.withgoogle.com/tasks/5378521694208000/

**Comment by PohSayKeong** [comment]
For the Impement improvements one am I allowed to claim it twice?

**Comment by PohSayKeong** [comment]
Or am I not supposed to claim it at all?

**Comment by niccokunzmann** [comment]
You can do the [github link](https://github.com/fossasia/flappy-svg/issues/7) but not the fossasia link. The github link is sufficiently difficult. The fossasia link is too easy for you. Better if we give this to a beginner so the beginner is able to contribute.

**Comment by PohSayKeong** [comment]
Oh ok sure, so I will remove the Fossasia one. However the GitHub link can only be opened when opened in new tab. I don't understand what's wrong with that one.

**Comment by PohSayKeong** [comment]
Hi, I will remove the both links as I did the GitHub one wrongly and I shouldn't be doing the Fossasia one. I will come back with the GitHub corner thing, is this lower difficulty or higher difficulty?

**Comment by PohSayKeong** [comment]
Hi, I have added the GitHub corner, the problem persists, it does not automatically redirect. Please tell me which task I should claim for this. While you are away, I will be working on creating packaged apps for the game.

**Comment by PohSayKeong** [comment]
As creating packaged apps requires icons and we do not have them yet, am I allowed to list this as an issue? As of right now I will only be using dummy icons.

**Comment by niccokunzmann** [comment]
Good idea. You can create "icons" as an issue. I will add some more ideas to your issue. You can also create a pull request with the empty icon and I will merge it so others know where to put the icon.

**Comment by PohSayKeong** [comment]
I added the icons folder together with this PR as I am assuming that you will be merging this request very soon? The only known issue here is the GitHub corner not automatically redirecting, other than that I think it's fine. Should I attempt to fix it or list it as an issue as well?

**Comment by niccokunzmann** [comment]
No - different topic, different pull request, different branch. It was
the same as with the backgrounds. Try to never put things together in
one pull request that do not belong together.

Am 23.12.2015 um 09:23 schrieb PohSayKeong:

**Comment by PohSayKeong** [comment]
Ok sure, I will revert this.

**Comment by niccokunzmann** [comment]
Feedback:
1. Comment in that issue that you want to do this issue https://github.com/fossasia/flappy-svg/issues/7
2. Claim the google codein task
3. The github edge should not be filled white.
4. The github edge should have the animation on mouseover.
5. In Inkscape refer to the source of the github edge to give them credit. Should be in the object properties.
6. Add a section in the Contributors.md named "Credits To" where you give credit to the creators of the github edge. (They give it for free so we should give them something back if we can and we can - reputation)

**Comment by PohSayKeong** [comment]
I am unable to claim the task right now unless I abandon my current one and then go back to it later, which is possible. I am having problems with the animation as in order for the corner to be In the startscreen I placed it in as svg using inkscape. Am I able to style it using css like how one styles html? If I place it in the html file, I would need to find a way to move it into the start screen.

**Comment by niccokunzmann** [comment]
If I look into the source code (linked in the issue #7 ), I think you can use javascript to start the animation. The object properties in Inkscape have a mouseover. This is my intuition - I do not know how to do this unless I do it :)

**Comment by PohSayKeong** [comment]
Sorry for making you to look in to the code, I made a css file to style the edge just like how one can do it in Html, works fine, added the contributors thing too,changed the color as well.

**Comment by niccokunzmann** [comment]
If I look at the animation, it is not waving. Can you correct that? The whole arm is flying off, up and down. I would like to see the waving. It is so welcoming. Can you remove the green thing or make it 99% transparent as well as the arm? If that does not work - please use the google codein colors.

**Comment by niccokunzmann** [comment]
I just looked - Iguess, it could be that the rotation center of the arm is in the wrong place. You can reach it if you double click on the arm - it is the small cross when the rotation arrows appear.

File: Contributors.md [hunk 1]
```diff
@@ -6,3 +6,9 @@ These people have conrtibuted to this game:
 - [Nicco Kunzmann](http://jugendprogrammiert.weebly.com/)
 - [Poh Say Keong] (https://github.com/PohSayKeong)
 
+
+
+
+Credits To:
+- [Lionel T] (http://codepen.io/elrumordelaluz/pen/XmygWX) (Github Edge)
```
Comment (niccokunzmann) [inline Contributors.md]: The real markdown syntax allows no space between "] (". Please change this to "]("

**Comment by niccokunzmann** [comment]
A different approach may be to edit the file directly - just a hint if nothing works out.

**Comment by PohSayKeong** [comment]
It only works when the code is 
<path d="M128.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0 123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2" fill="currentColor" style="transform-origin: 130px 106px;" class="octo-arm"></path>

Any other code will cause it to fail, when I move the arm it changes the code hence failing, I cannot move the arm in inkscape not even a little bit

**Comment by PohSayKeong** [comment]
path d="M128.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0 123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2" fill="currentColor" style="transform-origin: 130px 106px;" class="octo-arm"></path>

File: flappy.svg [hunk 2]
```diff
@@ -9361,6 +9361,32 @@
...
+           style="color:#ffffff;fill:#ff0000"
+           inkscape:connector-curvature="0"
+           d="m 380,0 46,46.000001 6,0 4.8,10.8 L 480,100 480,0 Z"
+           id="path5044" />
+        <path
+           inkscape:connector-curvature="0"
+           class="octo-arm"
+           style="color:#ffffff;fill:#008000"
+           d="m 431.32,43.600001 c -5.8,-3.72 -3.72,-7.76 -3.72,-7.76 1.2,-2.76 0.6,-4.4 0.6,-4.4 -0.52,-2.64 1.16,-0.92 1.16,-0.92 1.56,1.84 0.84,4.4 0.84,4.4 -1.04,4.12 2.04,5.84 3.56,6.36"
+           id="path5046" />
```
Comment (niccokunzmann) [inline flappy.svg]: The difference I spot is `style="transform-origin: 130px 106px;"`

    <path d="M158.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0
             123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2" 
          fill="currentColor" style="transform-origin: 130px 106px;" class="octo-arm">

**Comment by PohSayKeong** [comment]
Hi, thanks for that, I did mix them up, although I couldn't use 130px 106px, I experimented abit and managed to get it. It should wave now

**Comment by PohSayKeong** [comment]
Hi, it should be done now. I will abandon the current task of creating the app first and perhaps go back to it when the app is more complete. Please tell me which difficulty of implement improvement I should claim and I will go ahead and claim it there.

**Comment by PohSayKeong** [comment]
And the redirection thing from before is fixed too.

**Comment by niccokunzmann** [comment]
Hi, it looks better, good that you know how to change it now. 
- Can you change the rotation center to that of http://codepen.io/elrumordelaluz/pen/XmygWX
- Your link makes the page blank. Have a look at the fossasia link - it just opens the new page in a new tab and keeps the game page intact.

**Comment by PohSayKeong** [comment]
If I change the rotation center to the same one as http://codepen.io/elrumordelaluz/pen/XmygWX  it would not work. I don't think it would leave the page blank, I just downloaded from my own repo, it works fine.

**Comment by PohSayKeong** [comment]
I tried changing it to the Fossasia one, it makes the waving thing wave more in Firefox, and completely removes everything from Chrome. It worked fine for me on Chrome

**Comment by PohSayKeong** [comment]
Sorry my bad, I changed it to the one in http://codepen.io/elrumordelaluz/pen/XmygWX  and it works for both Chrome and Firefox now. The same problem still exists on IE though.

**Comment by PohSayKeong** [comment]
Weird, it doesn't work now, it worked for me just now.

**Comment by PohSayKeong** [comment]
Ok, it's done. It should work on both Chrome and Firefox now

**Comment by niccokunzmann** [comment]
It also works on opera. Cool! I really appreciate that waving : )

**Pull request merged**